### DISCUSSION TITLE
Extend Celery Result Database Backend

### DISCUSSION THREAD

**Discussion by adityar98** [celery/celery#8705] [Q&A]
I am using celery for performing a crawlin operation in this process I want to store a extra metadata information in the celery result table. I am using postgresql db as the backend. How can I add this extra column in the table and which should be updated be celery when creating a task.

**Comment by sdrabblescripta** [comment]
I ended up creating a separate table joined 1-1 with the primary result table. It's not ideal - there should be a way to extend the normal result table - but it works well. This was for django but I expect it would work similarly for plain celery.

**Reply by adityar98** [reply]
Thank you for that solution what I did was create the table in postgresql before initializing celery and adding in necessary columns which I populate using some generated value based on the values of other column so that I do not need to change the INSERT/UPDATE/DELETE of tasks in the celery. Using this approach it helped me to add metadata to the celery tasks
